modules/vector-search/pages/create-vector-search-index-ui.adoc

```diff
@@ -0,0 +1,73 @@
+= Create a Vector Search Index with the Web Console
+:page-topic-type: guide
+:description: You can create a Vector Search index with the Couchbase Server Web Console. 
+
+[abstract]
+{description}
+
+You must create a Vector Search index before you can xref:run-vector-search-ui.adoc[run a search] that supports vector comparisons.
+
+== Prerequisites
+
+* You have deployed the Search Service on a node in your database. 
+
+* You have a bucket with scopes and collections in your database. 
+
+* You have documents in a keyspace inside that bucket that contain vector embeddings. 
+
+* Your user account has the *Search Admin* role for the bucket where you want to create the index.  
+
+* You have logged in to the Couchbase Server Web Console. 
+
+== Procedure 
+
+To create a Vector Search index with the Couchbase Server Web Console: 
+
+. Go to *Search*.
+. Click btn:[Add Index].
+. In the *Index Name* field, enter a name for the Vector Search index. 
++
+[NOTE]
+====
+Your index name must start with an alphabetic character (a-z or A-Z). It can only contain alphanumeric characters (a-z, A-Z, or 0-9), hyphens (-), or underscores (_).
+
+For version 7.6 and later, your index name must be unique inside your selected bucket and scope. You cannot have 2 indexes with the same name inside the same bucket and scope.
+====
+
+. In the *Bucket* list, select the bucket that contains the documents you want to include in your index. 
+. Expand *Customize Index*. 
+. Select *Use non-default scope/collection(s)*.
+.. In the *Scope* list, select the scope that contains the documents you want to include in your index.
+. Expand *Mappings* and xref:create-type-mapping.adoc[create a new type mapping on a collection]:
+.. Click btn:[Add Type Mapping].
+.. In the *Collection* list, select the collection that contains the documents you want to include in your index. 
+.. Select *Only index specified fields*. 
+.. Click btn:[OK].
+. xref:search:create-child-field.adoc[] on the new collection type mapping with the following settings: 
+.. In the *Field* field, enter the name of the field in your documents that contains your vector embeddings.
++
+Vector embeddings should be an array of numbers. 
+.. In the *Type* list, select *vector*. 
+.. In the *Dimension* field, enter the total number of elements in the array that holds the vector embeddings for your documents.
++ 
+The Search Service supports arrays up to 2048 elements. 
+.. In the *Similarity metric* list, select the method to use to calculate the similarity between search term and Search index vectors.
++ 
+For more information, see xref:search:child-field-options-reference.adoc[].
+.. In the *Optimized for* list, select whether the Search Service should optimize Search queries for accuracy (*recall*) or speed (*latency*).
++
+For more information, see xref:search:child-field-options-reference.adoc[].
+.. Select *Index*.
+.. Click btn:[OK].
```

Comment:
```suggestion
. xref:search:create-child-field.adoc[] on the new collection type mapping. 
Make sure to include the vector settings *Dimension*, *Similarity metric*, and *Optimized for*.
+
For more information on child field settings, refer to xref:search:child-field-options-reference.adoc[].
```

Reply:
Just a suggestion - totally up to you. The info already exists on the other pages so it feels like it's just being duplicated in this task page.

Reply:
I'll have to think on this one. 🤔